app/views/panels/edit.html.haml

```diff
@@ -0,0 +1,71 @@
+- content_for(:page_title, "Edit Panel - #{@grant.name}")
+
+%h1
+  = @grant.name
+
+= render 'shared/grant_state', { grant: @grant }
+
+= render 'shared/grant_edit_tabbed_menu', { grant: @grant }
+
+%p
+  Lorem ipsum dolor sum introductory info about panels.
+
+= form_with(model: [@grant, @panel], method: :patch, local: true) do |f|
+  %h3
+    Meeting Information
+
+  .grid-x
+    .cell.small-12.medium-3
+      = f.label     :meeting_location
+    .cell.small-12.medium-9
+      = f.text_area :meeting_location, cols: 15, rows: 4
+  .grid-x
+    .cell.small-12.medium-3
+      = f.label       :meeting_link
+    .cell.small-12.medium-9
+      = f.url_field  :meeting_link, aria: { 'described-by': 'link-help' }
+      %p.help-text#link-help
+        e.g. secure (https) link to videoconference service
+  .grid-x
+    .cell.small-12.medium-3
+      = f.label :instructions
+    .cell.small-12.medium-9
+      = f.hidden_field  :instuctions, id: :panel_instructions
```

Comment:
for some reason the instructions are not saving when I test

Reply:
Fixed in c656db8